### PR TITLE
remove redundant pypi_deploy.sh script

### LINKED ISSUE #16: Use CircleCI deployment
instead of running at the end of the arch tests

### PR DESCRIPTION
**Pull request by Teyras**
Closes #16

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -91,6 +90,16 @@ jobs:
...
       - *doc
 
+  deploy:
+    docker:
+      - image: pritunl/archlinux:2017-07-08
+    working_directory: ~/cxflow-tensorflow
+    steps:
+      - *arch_deps
+      - checkout
+      - *install
```
Comment (FloopCZ) [inline .circleci/config.yml]: Is it necessary to install the package when building the wheel?

File: .circleci/config.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@ references:
       name: PyPI deploy
```
Comment (FloopCZ) [inline .circleci/config.yml]: Line 33 should say `run:`, not `deploy:` (I cannot comment in there)

File: .circleci/config.yml [hunk 3]
```diff
@@ -107,3 +116,11 @@ workflows:
...
             - test_ubuntu_rolling
             - test_archlinux
+      - deploy:
+          filters:
+            branches:
+              only: master
+          requires:
+            - test_ubuntu_latest
+            - test_ubuntu_rolling
+            - test_archlinux
```
Comment (petrbel) [inline .circleci/config.yml]: Similarly to Cognexa/cxflow#91 - I think we want to generate docs before the deployment

**Comment by petrbel** [comment]
@Teyras this PR closes #16, doesn't it?

**Comment by FloopCZ** [comment]
> @Teyras this PR closes #16, doesn't it?

@petrbel Nice try to automatically close the issue, but I actually don't think this works unless it is written in the PR description or one of the commits. :-)

**Comment by petrbel** [comment]
> @petrbel Nice try to automatically close the issue, but I actually don't think this works unless it is written in the PR description or one of the commits. :-)

This was actually a question :-)

**Comment by blazekadam** [comment]
Well its actually a question for you since you originally created that issue...

File: .circleci/config.yml (hunk 3 above)
Comment (blazekadam) [reply .circleci/config.yml]: Updated

**Comment by FloopCZ** [comment]
@blazekadam Not all comments were addressed and you have merged it prematurely. This is starting to be a rushed mess.

**Comment by blazekadam** [comment]
I overlooked the very first comment. Anyways, no harm done here. I appreciate your concerns yet it has to be done...

**Comment by blazekadam** [comment]
@FloopCZ as a matter of fact, the `deploy` directive is valid.

See https://circleci.com/docs/2.0/deployment_integrations/ 

That means the CI config is sound and complete.

**Comment by FloopCZ** [comment]
@blazekadam I know that it is valid, the only difference is that it only runs in the master branch. The branch is is already selected on the job level, so it is not necessary (and confusing) to check it on step level as well. (change `deploy:` to `run:` was actually @Teyras 's idea and he did it in the corresponding PR in cxflow).

**Pull request merged**